src/test/javascript/spec/helpers/mocks/directive/mock-translate-values.directive.ts

```diff
@@ -0,0 +1,6 @@
+import { Directive, Input } from '@angular/core';
+// tslint:disable-next-line:directive-selector
+@Directive({ selector: '[translateValues]' })
+export class MockTranslateValuesDirective {
+    @Input('translateValues') data: any;
+}
```

Comment:
Just a question, does this imply?

Reply:
Imply what?

Reply:
ah, I mean, why is this added?

Reply:
There was an error for these tests in which the here mentioned selector was not found after we mocked everything. So this is the fix for that.
I moved it to a dedicated mock file since previously it was in one specialised test class and this caused issues as well.